plugins/woocommerce/tests/php/src/Internal/Logging/RemoteLoggerTest.php

```diff
@@ -109,16 +109,7 @@ public function remote_logging_disallowed_provider() {
 					'setup'     => function () {
 						$version = WC()->version;
 						// Next major version. (e.g. 9.0.1 -> 10.0.0).
-						$next_version = implode(
-							'.',
-							array_map(
-								function ( $n, $i ) {
-									return 0 === $i ? $n + 1 : 0;
-								},
-								explode( '.', $version ),
-								array_keys( explode( '.', $version ) )
-							)
-						);
+						$next_version = sprintf( '%d.0.0', explode( '.', $version )[0] + 1 );
```

Comment:
Nice simplification!

Reply:
BTW, you can add a third argument to `explode` call to reduce the number of split operations it performs.

Reply:
Done in 45df1fef5805d0eb7ff6590dccfc2ae676bbbf3a.